package.json

```diff
@@ -4,9 +4,12 @@
   "description": "A platform where people upload add books ",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "nyc --reporter=html --reporter=text-summary mocha -r esm test/**/*.test.js",
+    "test-watch": "nodemon --exec npm test",
     "lint": "eslint --fix --ext .js .",
-    "start": "node -r esm ."
+    "start": "node -r esm .",
+    "coveralls": "nyc npm run test-build && nyc report --reporter=text-lcov | coveralls",
```

Comment:
Why do you have a test-build on this line?. I don't see a test build in amongst your scripts?